src/main/java/frc/robot/Constants.java

```diff
@@ -74,11 +66,41 @@ public static class CoralScorerConstants {
 
   // Intake Constants
   public static class IntakeConstants {
-    public static final double kSpeed = 0.5;
+    public static final double kIntakeSpeed = 0.5;
+    public static final double kConveyorSpeed = 0.5;
+    public static final double kPivotSpeed = 0.2;
+    public static final Angle kPivotRetractAngle = Degrees.of(90); // FIXME: Put actual value
+    public static final Angle kPivotExtendAngle =
+        Degrees.of(0); // FIXME: Might have to be negative to reach over bumper
+    public static final Angle kPivotTolerance = Degrees.of(3);
+    public static final double kPivotP = 100.0;
+    public static final double kPivotI = 0.0;
+    public static final double kPivotD = 0.0;
+    public static final double kPivotG = 0.0;
+
+    public static final double kPivotV = 7.29; // 7.20;
+    public static final double kPivotA = 0.03; // 0.03;
+    public static final GravityTypeValue kPivotGravityType = GravityTypeValue.Arm_Cosine;
+
+    public static final double kGearing = 60;
+    public static final double kSensorToMechanism = 60;
+    public static final Distance kLength = Feet.of(1);
+    public static final Mass kMass = Pounds.of(8);
+    public static final MomentOfInertia kMOI =
+        KilogramSquareMeters.of(
+            SingleJointedArmSim.estimateMOI(kLength.in(Meters), kMass.in(Kilograms)));
+    public static final AngularVelocity kMotionMagicCruiseVelocity =
+        DegreesPerSecond.of(30); // RevolutionsPerSecond.of(10);
+    public static final AngularAcceleration kMotionMagicAcceleration =
+        kMotionMagicCruiseVelocity.times(Hertz.of(5));
+    public static final double kMotionMagicJerk = 10.0;
+
+    public static final double armZero = 0.0;
   }
 
+  // Elevator Constants
   public static class ElevatorConstants {
-    public static final Distance kL0Height = Meters.of(0.252);
+    public static final Distance kL0Height = Meters.of(0);
```

Comment:
why was this changed in the intake branch?

Reply:
Looks like this was changed in both this and the Elevator PR, so we should be able to leave it alone